client/lib/purchases/index.js

```diff
@@ -372,6 +377,10 @@ function purchaseType( purchase ) {
 		return i18n.translate( 'Premium Theme' );
 	}
 
+	if ( isConciergeSession( purchase ) ) {
+		return i18n.translate( 'Concierge Support' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Priority &amp; Concierge Support' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).